pkg/apis/componentconfig/edgecore/v1alpha1/types.go

```diff
@@ -336,6 +336,18 @@ type EventBus struct {
 	// MqttServerExternal indicates external mqtt broker url
 	// default "tcp://127.0.0.1:1883"
 	MqttServerExternal string `json:"mqttServerExternal,omitempty"`
+	// MqttSubClientID indicates mqtt subscribe ClientID
+	// default ""
+	MqttSubClientID string `json:"mqttSubClientID"`
```

Comment:
```suggestion
	MqttSubClientID string `json:"mqttSubClientID,omitempty"`
```
Same as below.

Reply:
if add omitempty, if we use `edgecore --defaultconfig` cmd to generate config file, these fields will be empty, and just contain below things.
```
eventBus:
    enable: true
    eventBusTLS:
      enable: false
      tlsMqttCAFile: /etc/kubeedge/ca/rootCA.crt
      tlsMqttCertFile: /etc/kubeedge/certs/server.crt
      tlsMqttPrivateKeyFile: /etc/kubeedge/certs/server.key
    mqttMode: 2
    mqttQOS: 0
    mqttRetain: false
    mqttServerExternal: tcp://127.0.0.1:1883
    mqttServerInternal: tcp://127.0.0.1:1884
    mqttSessionQueueSize: 100
```

Reply:
seems not obvious for users to set username and password

Reply:
> if add omitempty, if we use edgecore --defaultconfig cmd to generate config file, these fields will be empty,

Make sense, we need ensure them printed in default config and omit in mincaonfig.

Reply:
even these fields are empty string, also printed
```
eventBus:
    enable: true
    eventBusTLS:
      enable: false
      tlsMqttCAFile: /etc/kubeedge/ca/rootCA.crt
      tlsMqttCertFile: /etc/kubeedge/certs/server.crt
      tlsMqttPrivateKeyFile: /etc/kubeedge/certs/server.key
    mqttMode: 2
    mqttPassword: ""
    mqttPubClientID: ""
    mqttQOS: 0
    mqttRetain: false
    mqttServerExternal: tcp://127.0.0.1:1883
    mqttServerInternal: tcp://127.0.0.1:1884
    mqttSessionQueueSize: 100
    mqttSubClientID: ""
    mqttUsername: ""
```